run-tests-with-build-optaplanner-quickstarts-parent/optaplanner-quickstarts-project-sources-test/pom.xml

```diff
@@ -51,23 +59,78 @@
         <artifactId>maven-invoker-plugin</artifactId>
         <configuration>
           <parallelThreads>1</parallelThreads>
-          <!--
-          script deciding if included project is:
-            executed (either script missing or returning true)
-              or
-            skipped (when returns false or throws error) -->
-          <selectorScript>${maven.modules.resolution.selector.script.name}</selectorScript>
-          <pomIncludes>
-            <!-- includes everything, filtering is done in resolve-includes.groovy script
-            which adds invoker-run.groovy script containing 'return false' near each pom.xml
-            that should be excluded.-->
-            <pomInclude>**/pom.xml</pomInclude>
-          </pomIncludes>
-          <pomExcludes>
-            <!-- Excludes work still as expected. -->
-          </pomExcludes>
         </configuration>
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>non-native</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
If really needed - don't use activeByDefault, it's prone to mistake. It's better to use property name `!native` if there are just 2 variants.

Reply:
Fixed d79f3f5a